recipes/poco/all/patches/1.10.0.patch

```diff
@@ -0,0 +1,73 @@
+--- cmake/PocoMacros.cmake
++++ cmake/PocoMacros.cmake
+@@ -278,7 +278,7 @@
+ 
+ if (MSVC)
+ # install the targets pdb
+-  POCO_INSTALL_PDB(${target_name})
++  # POCO_INSTALL_PDB(${target_name})
+ endif()
+ 
+ endmacro()
+--- Crypto/CMakeLists.txt
++++ Crypto/CMakeLists.txt
+@@ -21,7 +21,7 @@
+     DEFINE_SYMBOL Crypto_EXPORTS
+     )
+ 
+-target_link_libraries(Crypto PUBLIC Poco::Foundation OpenSSL::SSL OpenSSL::Crypto)
++target_link_libraries(Crypto PUBLIC Poco::Foundation OpenSSL::OpenSSL)
+ target_include_directories(Crypto
+     PUBLIC
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+--- Data/SQLite/CMakeLists.txt
++++ Data/SQLite/CMakeLists.txt
+@@ -7,7 +7,7 @@
+ POCO_HEADERS_AUTO( SQLITE_SRCS ${HDRS_G})
+ 
+ if (POCO_UNBUNDLED)
+-	find_package(SQLite3 REQUIRED)
```

Comment:
better fix sqlite recipe?

Reply:
Indeed. https://cmake.org/cmake/help/latest/module/FindSQLite3.html

Reply:
Fixed in d04db99